cmd/collect_signals/README.md

```diff
@@ -0,0 +1,205 @@
+# Signal Collector
+
+This tool is used to collect signal data for a set of project repositories for
+generating a criticality score.
+
+The input of this tool could by the output of the `enumerate_github` tool.
+
+The output of this tool is used as an input for the `scorer` tool, or for input
+into a data analysis system such as BigQuery. 
+
+## Example
+
+```shell
+$ export GITHUB_TOKEN=ghp_x  # Personal Access Token Goes Here 
+$ export ...
```

Comment:
What is ... referring to? can we omit it for simplicity?

Reply:
This was meant for GCP. Fixed.